### ISSUE TITLE
Allow different number of threads for lanczos downscaling

### ISSUE THREAD

**Issue by utack** [k4yt3x/video2x#575]
## Description
It would be nice to have the step "Lanczos downscaling frames" running with a different thread number than the upscaling process
The Lanczos downscaler seems to be purely CPU bound, and in my case takes 3-4x longer than upscaling only using 1  CPU core

Thank you for consideration

**Comment by k4yt3x** [comment]
Someone has already brought this up earlier. The next version of v2x will be multi-processed and Lanczos operations will run in parallel. I just didn't bother adding it to the current version as my dev environment for 4.8 is already nonexistent.
